apps/pom.xml

```diff
@@ -299,7 +303,7 @@
 				<groupId>org.hsqldb</groupId>
 				<artifactId>hsqldb</artifactId>
 				<!-- The 2.5.x series is the last to support Java 8. -->
-				<version>2.5.2</version>
+				<version>2.7.0</version>
```

Comment:
Should reference the hsqldb.version above (and can remove the comment about java 8 here)

Reply:
I think the ask here is to replace 2.7.0 with ${hsqldb.version}

also I think the comment on line 305 can just be removed entirely.

Reply:
Updated